cfgov/unprocessed/js/modules/transition/AlphaTransition.js

```diff
@@ -43,28 +45,37 @@ function AlphaTransition( element ) {
   }
 
   /**
-   * Remove all transition classes.
+   * Remove all transition classes, if transition is initialized.
+   * @returns {boolean}
+   *   True, if the element's CSS classes were touched, false otherwise.
    */
   function remove() {
-    _baseTransition.remove();
+    return _baseTransition.remove();
   }
 
   /**
    * Add a "transition-duration: 0" utility CSS class.
+   * @returns {boolean}
+   *   Whether the element's CSS classes where touched or not.
+   *   False means the transition has not been initialized.
    */
   function animateOn() {
-    _baseTransition.animateOn();
+    return _baseTransition.animateOn();
   }
 
   /**
    * Remove a "transition-duration: 0" utility CSS class.
+   * @returns {boolean}
+   *   Whether the element's CSS classes where touched or not.
+   *   False means the transition has not been initialized.
    */
   function animateOff() {
-    _baseTransition.animateOff();
+    return _baseTransition.animateOff();
```

Comment:
Yeah, I guess my worry is that someone wouldn't know that you only are returning a boolean to make the code more testable. The original intent might be lost and someone might start using those boolean return values in ways you didn't intend.

Also, the transitions all seem to be repeating the same methods,  why not make them mixins?